portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -20,11 +20,13 @@
 public final class Comment {
 
   private final long id;
+  private final String name;
```

Comment:
Nit: Not a big deal but it will be better if you could change it to userName for cellularity.

Reply:
changed instances of 'name' in this context to 'username'